styles/components/position-hero.scss

```diff
@@ -0,0 +1,13 @@
+.position-hero {
+  position: relative;
+  width: 100%;
+  height: 40vh;
+  background: $c-position;
+
+  &__title {
+    position: absolute;
+    bottom: 50%;
```

Comment:
Unless I have a need to pin an item to the bottom edge, I lend a preference to `top`.